gnosis/safe-cmm-app-react
# Feature/#7/price component

# Description

- Added Price input component
- Refactored other components into reusable parts
- Added storybook controls and docs

![screenshot_2020-08-19_15-30-29](https://user-images.githubusercontent.com/43217/90696327-e9615880-e230-11ea-8653-b1f0bf0d1f90.png)


# To Test
1. Start storybook server `npm run storybook`
2. Visit `http://localhost:6006/?path=/docs/priceinput--default`

- [ ] Verify Price input matches designs https://projects.invisionapp.com/share/VXXWLEE7534#/screens/427462296
- [ ] Should have label and tooltip
- [ ] Input should only accept Numbers

# Background

## TODO:
Tooltip not yet match designs. Will work on that next.
